### PR TITLE
Send info about absent users once a day in common channel

### LINKED ISSUE #64: Send info about absent users once a day in common channel
Bot should collect info about all absent users and send one message once a day in common channel. In the message users should be group by the reason of absence and message should be transformed depends on users number in each category.
Example:
```
Сегодня 8 ноября. 
Из дома работают: @имя, @имя, @имя
Отсутствует по болезни: @имя 
В отпуске: @имя
```

### PR DESCRIPTION
**Pull request by BehindLoader**
close #64 
# Please make sure all below checkboxes in the Mandatory section have been checked before submitting your PR (also don't forget to pay attention to the Depending on Circumstances section)

## Mandatory

- [x] The commit message is an imperative and starts with a capital letter (for example, `Add something`, `Remove something`, `Fix something`, etc.).
- [x] I made sure I hadn't put a period at the end of the the commit message(s).

## Depending on Circumstances

Some of the items in the section become mandatory if you are a first-time contributor and/or your changes are relevant to the items.

- [x] Adding a new parameter I didn't forget to reflect it in `README.md`.
- [x] Modifying a list of constants or imports I didn't brake an alphabetical order.
- [ ] (for first-time contributors) One of the commits in the pull request adds me to the **Acknowledgements** section in `AUTHORS.md`.

### CODE REVIEW

File: src/viva.js [hunk 1]
```diff
@@ -424,6 +425,60 @@ module.exports = async (robot) => {
     return false
   }
 
+  /*
+   * TODO
+   *
+   * @param {Robot} robot - Robot instance.
+   */
+  function dailyReport (robot) {
```
Comment (eugulixes) [inline src/viva.js]: Let's rename it into `prepareDailyReport`.

File: README.md [hunk 2]
```diff
@@ -38,6 +38,7 @@ The script can be configured via the following environment variables (called par
 | `MAXIMUM_LENGTH_OF_WAIT`      | The maximum number of days handling of each request may take. | 7 |
 | `MINIMUM_DAYS_BEFORE_REQUEST` | The minimum number of days before the target date an employee is allowed to apply for a leave. | 7 |
 | `VIVA_REMINDER_SCHEDULER` | Allows specifying the frequency with which the script checks if some of the users are awaiting a reply or back after leave. If the check succeeds, the script sends reminders (either to the channel specified via `LEAVE_COORDINATION_CHANNEL` or directly to users, depending on the particular reminder). The value of this parameter must follow the [Cron Format](https://github.com/node-schedule/node-schedule#cron-style-scheduling). | `0 0 7 * * *` |
+| `VIVA_REPORT_SCHEDULER` | Allows specifying the frequency with which the script checks if did the user remind you about leaving. If the check succeeds, the script sends reminders (either to the channel specified via `LEAVE_COORDINATION_CHANNEL` or directly to users, depending on the particular reminder). The value of this parameter must follow the [Cron Format](https://github.com/node-schedule/node-schedule#cron-style-scheduling). | `0 0 7 * * *` |
```
Comment (polina-popova) [inline README.md]: The description doesn't fit the parameter - it doesn't depend on users reminder and doesn't send to the leave-coordination channel, for example.

File: README.md [hunk 3]
```diff
@@ -38,6 +38,7 @@ The script can be configured via the following environment variables (called par
 | `MAXIMUM_LENGTH_OF_WAIT`      | The maximum number of days handling of each request may take. | 7 |
 | `MINIMUM_DAYS_BEFORE_REQUEST` | The minimum number of days before the target date an employee is allowed to apply for a leave. | 7 |
 | `VIVA_REMINDER_SCHEDULER` | Allows specifying the frequency with which the script checks if some of the users are awaiting a reply or back after leave. If the check succeeds, the script sends reminders (either to the channel specified via `LEAVE_COORDINATION_CHANNEL` or directly to users, depending on the particular reminder). The value of this parameter must follow the [Cron Format](https://github.com/node-schedule/node-schedule#cron-style-scheduling). | `0 0 7 * * *` |
+| `VIVA_REPORT_SCHEDULER` | Allows you to specify the frequency with which the script checks the status of the user. If the check is successful, the script sends in `general` reminder. The value of this parameter should correspond to [Cron Format] (https://github.com/node-schedule/node-schedule#cron-style-scheduling). | `0 0 11 * * *` |
```
Comment (polina-popova) [inline README.md]: ```Allows specifying the frequency with which script checks status of all users and forms the list of absence users (start vacation/work from home day) or users who come at work first day after a break (back from vacation), sorted by the reason, and sends it to 'general' channel. The value of this parameter should correspond to [Cron Format] (https://github.com/node-schedule/node-schedule#cron-style-scheduling).```

File: src/viva.js [hunk 4]
```diff
@@ -430,6 +431,63 @@ module.exports = async (robot) => {
     return false
   }
 
+  /*
+   * Sends to the general channel information about the status of users.
+   *
+   * @param {Robot} robot - Robot instance.
+   */
```
Comment (polina-popova) [inline src/viva.js]: `Sends` should be without `s` ending.

**Pull request merged**